internal/venice-test-common/src/integrationtest/java/com/linkedin/venice/controller/TestParentControllerWithMultiDataCenter.java

```diff
@@ -385,4 +386,66 @@ public void testEnableActiveActiveReplicationSchema() {
+    try (ControllerClient parentControllerClient = new ControllerClient(clusterName, parentControllerURLs);
+        ControllerClient dc0Client =
+            new ControllerClient(clusterName, childDatacenters.get(0).getControllerConnectString());
+        ControllerClient dc1Client =
+            new ControllerClient(clusterName, childDatacenters.get(1).getControllerConnectString())) {
+
+      NewStoreResponse newStoreResponse =
+          parentControllerClient.retryableRequest(5, c -> c.createNewStore(storeName, "", "\"string\"", "\"string\""));
+      Assert.assertFalse(
+          newStoreResponse.isError(),
+          "The NewStoreResponse returned an error: " + newStoreResponse.getError());
+
+      List<ControllerClient> childControllerClients = new ArrayList<>();
+      childControllerClients.add(dc0Client);
+      childControllerClients.add(dc1Client);
+      emptyPushToStore(parentControllerClient, childControllerClients, storeName, 1);
+      // Rollback should fail since there is no backup version
+      ControllerResponse response = parentControllerClient.rollbackToBackupVersion(storeName);
+      Assert.assertTrue(response.isError());
+
+      emptyPushToStore(parentControllerClient, childControllerClients, storeName, 2);
+      // Should roll back to version 1
+      response = parentControllerClient.rollbackToBackupVersion(storeName);
+      Assert.assertFalse(response.isError());
+      for (ControllerClient childControllerClient: childControllerClients) {
+        TestUtils.waitForNonDeterministicAssertion(10, TimeUnit.SECONDS, false, true, () -> {
+          StoreResponse storeResponse = childControllerClient.getStore(storeName);
+          Assert.assertFalse(storeResponse.isError());
+          StoreInfo storeInfo = storeResponse.getStore();
+          Assert.assertEquals(storeInfo.getCurrentVersion(), 1);
+        });
+      }
+    }
+  }
+
+  private void emptyPushToStore(
+      ControllerClient parentControllerClient,
+      List<ControllerClient> childControllerClients,
+      String storeName,
+      int expectedVersion) {
+    VersionCreationResponse vcr = parentControllerClient.emptyPush(storeName, Utils.getUniqueString("empty-push"), 1L);
+    assertEquals(
+        vcr.getVersion(),
+        expectedVersion,
+        "requesting a topic for a push should provide version number " + expectedVersion);
+    for (ControllerClient childControllerClient: childControllerClients) {
+      TestUtils.waitForNonDeterministicAssertion(10, TimeUnit.SECONDS, false, true, () -> {
+        StoreResponse storeResponse = childControllerClient.getStore(storeName);
+        Assert.assertFalse(storeResponse.isError());
+        StoreInfo storeInfo = storeResponse.getStore();
+        Assert.assertEquals(storeInfo.getCurrentVersion(), expectedVersion);
+      });
+    }
+    parentControllerClient.deleteKafkaTopic(Version.composeKafkaTopic(storeName, expectedVersion));
```

Comment:
Just curious why we need to delete kafka topic after empty push, for specific reason?

Reply:
At the very beginning I wanted to ensure the first empty push's version topic is truncated so that it won't block the second empty push. After checking the code again, I realize that the explicit truncation is not needed. When the second empty push runs, it will query previous job status and truncate topic: https://github.com/linkedin/venice/blob/4fa7bc6e9a627982f763dddf9dfd95f0737b5024/services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceParentHelixAdmin.java#L1247 The second push job won't be blocked. I'll remove line 449 in the test.